tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -72,6 +72,38 @@ def encode(self) -> bytes:
         return buffer.data
 
 
+class MyH3Connection(H3Connection):
```

Comment:
We should probably have a more descriptive name for this, unless the "My" prefix is a convention.

Reply:
Done.